08-service/03-internal-service.yml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: simple-service
+  name: simple-internal-service
```

Comment:
why renamaing it?

Reply:
it wasn't aligned the doc, reference tot `simple-internal-service` but name was the above.

Reply:
🆗